Jenkins - Creating sponsor thanks & appreciation post

kmartens27 opened the pull request:
This is a pull request to add a blog post highlighting and thanking our many sponsors over the year.

MarkEWaite commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,49 @@
+image:/images/sponsors/netlify.png[Netlify, width=30%]
+image:/images/sponsors/linux-foundation.png[The Linux Foundation, width=27%]
+
+Massive thanks to link:https://www.algolia.com/[Algolia] for providing site search for the primary Jenkins and plugins sites.
```
The phrasing felt odd to me.  Would an extra word help here?
```suggestion
Massive thanks to link:https://www.algolia.com/[Algolia] for providing site search for the primary Jenkins documentation and plugins sites.
```

kmartens27 replied:
yes, definitely thanks @MarkEWaite!

MarkEWaite commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,49 @@
+authors:
+- kmartens27
+description: >
+    The Jenkins project shares gratitude to our amazing sponsors.
```
I thought that two space indentation was typical here.
```suggestion
  The Jenkins project shares gratitude to our amazing sponsors.
```

MarkEWaite reviewed (approved):
I'd like to review the page in the preview before it is merged.  Text looks good.  Optional comments for your consideration

timja commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,49 @@
+
+We also want to recognize and thank link:https://github.com/[Github], link:https://jfrog.com/[JFrog], link:https://www.atlassian.com/[Atlassian], link:https://www.linuxfoundation.org/[Linux Foundation], link:https://www.netlify.com/[Netlify], and link:https://1password.com/[1Password] for providing tools to track Jenkins development.
+
+image:/images/sponsors/github.png[Github, width=30%]
```
```suggestion
image:/images/sponsors/github.png[GitHub, width=30%]
```

timja commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,49 @@
+We also want to recognize and thank link:https://github.com/[Github], link:https://jfrog.com/[JFrog], link:https://www.atlassian.com/[Atlassian], link:https://www.linuxfoundation.org/[Linux Foundation], link:https://www.netlify.com/[Netlify], and link:https://1password.com/[1Password] for providing tools to track Jenkins development.
+
+image:/images/sponsors/github.png[Github, width=30%]
+image:/images/sponsors/jfrog.png[Jfrog, width=15%]
```
```suggestion
image:/images/sponsors/jfrog.png[JFrog, width=15%]
```

timja commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,49 @@
+image:/images/sponsors/github.png[Github, width=30%]
+image:/images/sponsors/jfrog.png[Jfrog, width=15%]
+image:/images/sponsors/atlassian.png[Atlassian, width=40%]
+image:/images/sponsors/1password.png[1 Password, width=30%]
```
```suggestion
image:/images/sponsors/1password.png[1Password, width=30%]
```

timja commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,49 @@
+image:/images/sponsors/atlassian.png[Atlassian, width=40%]
+image:/images/sponsors/1password.png[1 Password, width=30%]
+image:/images/sponsors/netlify.png[Netlify, width=30%]
+image:/images/sponsors/linux-foundation.png[The Linux Foundation, width=27%]
```
```suggestion
image:/images/sponsors/linux-foundation.png[Linux Foundation, width=27%]
```

timja commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,49 @@
+image:/images/sponsors/digital-ocean.png[Digital Ocean, width=30%]
+image:/images/sponsors/osuosl.png[OSU Open Source Labs, width=30%]
+
+We also want to recognize and thank link:https://github.com/[Github], link:https://jfrog.com/[JFrog], link:https://www.atlassian.com/[Atlassian], link:https://www.linuxfoundation.org/[Linux Foundation], link:https://www.netlify.com/[Netlify], and link:https://1password.com/[1Password] for providing tools to track Jenkins development.
```
```suggestion
We also want to recognize and thank link:https://github.com/[GitHub], link:https://jfrog.com/[JFrog], link:https://www.atlassian.com/[Atlassian], link:https://www.linuxfoundation.org/[Linux Foundation], link:https://www.netlify.com/[Netlify], and link:https://1password.com/[1Password] for providing tools to track Jenkins development.
```

timja reviewed (commented):
correcting brand names

kmartens27 commented:
Thanks for catching those @timja, I appreciate the review!

MarkEWaite commented:
Layout of the images needs work.  The sponsors won't approve of the lack of spacing between the images in the presentation.  It places the logo of one vendor too close to the logo of other vendors.

The [home page](https://www.jenkins.io/) uses some technique to provide spacing between vendor logos, but I'm not sure how to apply that technique (from a HAML file) to the adoc file we're using here.

MarkEWaite commented:
I've made each of the sponsor images also be a hyperlink to the sponsor web site so that readers who click a sponsor image will be taken to the sponsor site.

The more I look at it, the more I think that we want some text to better describe the contributions from each of the sponsors.  However, that's not context that @kmartens27 has.  I'd like to hold on merging this until tomorrow so that I can have some time during the early hours tomorrow morning to propose some text additions that are more specific for each of the sponsors.  That text will then be a way to separate the logos from each other and show that we are aware of the details of the contributions by each sponsor.

lemeurherve commented:
With the permission of @kmartens27 I've pushed compressed images optimized with compressor.io

![image](https://user-images.githubusercontent.com/91831478/203664833-a7d60b8c-517d-44c9-a6cd-b410da53435b.png)

lemeurherve commented:
> The more I look at it, the more I think that we want some text to better describe the contributions from each of the sponsors. 

How about linking each sponsor name with the last corresponding sponsor blog post?

MarkEWaite commented:
> > The more I look at it, the more I think that we want some text to better describe the contributions from each of the sponsors.
> 
> How about linking each sponsor name with the last corresponding sponsor blog post? Or adding next to their URL the title and link to these? Quick and easy, while allowing readers to get more info, and these blog posts more exposure.

Good idea.  I'm not sure that we have a blog post for each of the sponsors, but including links to the existing blog posts makes good sense to me for those cases where we have a blog post

dduportal reviewed (commented):
This is only a comment, I would like to add some sponsors. If it make sense, let me know and I'll make suggestions.

- Datadog (we have full open access to their platform with no restriction)
- Docker (I realize we did not mark them on the jenkins.io website, but they've let us access to the enterprise account for the Jenkisn images with many benefits for us including no more rate limits and better management and security)
- PagerDuty (They recently renewed the sponsorships thanks to Mark)
- Fastly (CDN for the jenkins.io websites, and a pretty efficient one)
- GitHub (we have access to a lot of paid feature as far as I can tell)
- Microsoft Azure (most of the infrastructure runs on it. I'm not sure if they sponsor us or if we pay them money from the CDF. Maybe mentioning Microsoft for GitHub and Azure so that would settle it?)

kmartens27 commented:
I added in a preview image that @alyssat had created, and added Digital Ocean to the funding section since they recently donated a bunch to us.

Thanks to @MarkEWaite, @lemeurherve, @alyssat, @dduportal, and @timja for all of your review, feedback, and assistance!

timja commented:
> Microsoft Azure (most of the infrastructure runs on it. I'm not sure if they sponsor us or if we pay them money from the CDF. Maybe mentioning Microsoft for GitHub and Azure so that would settle it?)

They no longer sponsor us

zbynek commented:
Maybe the algolia logo could be cropped to match the others (has a lot more padding).

kmartens27 commented:
Thanks very much @zbynek for that! I've cropped that image and compressed it as @lemeurherve had done previously. @alyssat also updated the opengraph image to be used so it has more of our sponsors.

dduportal commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,52 @@
+Sponsorship of the Jenkins project is key to the maintenance and development of Jenkins.
+There are different ways in which our sponsor organizations support the Jenkins project, such as infrastructure, necessary development tools, funding, and providing mirrors to connect Jenkins to the community.
+
+For supporting Jenkins distribution, build, test, and deployment infrastructure, we would like to thank link:https://jfrog.com/[JFrog], https://aws.amazon.com/[AWS], link:https://www.oracle.com/[Oracle], link:https://www.digitalocean.com/[Digital Ocean], and link:https://osuosl.org/[OSU Open Source Labs].
```
```suggestion
For supporting Jenkins distribution, build, test, and deployment infrastructure, we would like to thank link:https://jfrog.com/[JFrog], https://aws.amazon.com/[AWS], link:https://www.oracle.com/[Oracle], link:https://www.digitalocean.com/[Digital Ocean], link:https://www.datadoghq.com[Datadog], link:https://www.docker.com/[Docker], link:https://www.pagerduty.com/[PagerDuty], link:https://www.fastly.com[Fasty], link:https://github.com[GitHub] and link:https://osuosl.org/[OSU Open Source Labs].
```

dduportal commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,52 @@
+image:/images/sponsors/aws.png[AWS, width=25%, link="https://aws.amazon.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/oracle.png[Oracle, width=35%, link="https://www.oracle.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/digital-ocean.png[Digital Ocean, width=30%, link="https://www.digitalocean.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/osuosl.png[OSU Open Source Labs, width=30%, link="https://osuosl.org/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
```
```suggestion
image:/images/sponsors/osuosl.png[OSU Open Source Labs, width=30%, link="https://osuosl.org/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/datadog.png[Datadog, width=30%, link="https://www.datadoghq.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/docker.png[Docker, width=30%, link="https://www.docker.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/pagerduty.png[PagerDuty, width=30%, link="https://www.pagerduty.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/github.png[GitHub, width=30%, link="https://github.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/fastly.png[Fastly, width=30%, link="https://www.fastly.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
```

=> incoming commit with the images

lemeurherve reviewed (changes requested):
The CloudBees logo on the OG image is in too bad state to be published

lemeurherve reviewed (commented):
CloudBees logo and OG image updated

kmartens27 commented:
Thanks very much @lemeurherve, would you be able to review everything once more?

lemeurherve commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,56 @@
+Sponsorship of the Jenkins project is key to the maintenance and development of Jenkins.
+There are different ways in which our sponsor organizations support the Jenkins project, such as infrastructure, necessary development tools, funding, and providing mirrors to connect Jenkins to the community.
+
+For supporting Jenkins distribution, build, test, and deployment infrastructure, we would like to thank link:https://jfrog.com/[JFrog], https://aws.amazon.com/[AWS], link:https://www.oracle.com/[Oracle], link:https://www.digitalocean.com/[Digital Ocean], link:https://www.datadoghq.com[Datadog], link:https://www.docker.com/[Docker], link:https://www.pagerduty.com/[PagerDuty], link:https://www.fast ...
+
+image:/images/sponsors/jfrog.png[JFrog, width=15%, link="https://jfrog.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/aws.png[AWS, width=25%, link="https://aws.amazon.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/oracle.png[Oracle, width=35%, link="https://www.oracle.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/digital-ocean.png[Digital Ocean, width=30%, link="https://www.digitalocean.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/osuosl.png[OSU Open Source Labs, width=30%, link="https://osuosl.org/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/datadog.png[Datadog, width=30%, link="https://www.datadoghq.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/docker.png[Docker, width=30%, link="https://www.docker.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/pagerduty.png[PagerDuty, width=30%, link="https://www.pagerduty.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/fastly.png[Fastly, width=30%, link="https://www.fastly.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+
+We also want to recognize and thank link:https://github.com/[GitHub], link:https://jfrog.com/[JFrog], link:https://www.atlassian.com/[Atlassian], link:https://www.linuxfoundation.org/[Linux Foundation], link:https://www.netlify.com/[Netlify], and link:https://1password.com/[1Password] for providing tools to track Jenkins development.
+
+image:/images/sponsors/github.png[GitHub, width=30%, link="https://github.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/jfrog.png[JFrog, width=15%, link="https://jfrog.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/atlassian.png[Atlassian, width=40%, link="https://www.atlassian.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/1password.png[1Password, width=30%, link="https://1password.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/netlify.png[Netlify, width=30%, link="https://www.netlify.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/linux-foundation.png[Linux Foundation, width=27%, link="https://www.linuxfoundation.org/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+
+Massive thanks to link:https://www.algolia.com/[Algolia] for providing site search for the primary Jenkins documentation and plugins sites.
+
+image:/images/sponsors/algolia.png[Algolia, width=40%, link="https://www.algolia.com/"]
+
+We also want to share deep gratitude for operational funding from link:https://cd.foundation/[CD Foundation], link:https://www.cloudbees.com/[CloudBees], link:https://aws.amazon.com/[AWS], and link:https://www.digitalocean.com/[Digital Ocean].
+Without funding, the community would have less opportunities to work on and develop Jenkins.
+
+image:/images/sponsors/cdf.png[CD Foundation, link="https://cd.foundation/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/cloudbees.png[CloudBees, link="https://www.cloudbees.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/aws.png[AWS, link="https://aws.amazon.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
+image:/images/sponsors/digital-ocean.png[Digital Ocean, width=30%, link="https://www.digitalocean.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
```
```suggestion
For supporting Jenkins distribution, build, test, and deployment infrastructure, we would like to thank link:https://jfrog.com/[JFrog], https://aws.amazon.com/[AWS], link:https://www.oracle.com/[Oracle], link:https://www.digitalocean.com/[DigitalOcean], link:https://www.datadoghq.com[Datadog], link:https://www.docker.com/[Docker], link:https://www.pagerduty.com/[PagerDuty], link:https://www.fastly.com[Fasty], link:https://github.com[GitHub] and link:https://osuosl.org/[OSU Open Source Labs].

image:/images/sponsors/jfrog.png[JFrog, width=15%, link="https://jfrog.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/aws.png[AWS, width=25%, link="https://aws.amazon.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/oracle.png[Oracle, width=35%, link="https://www.oracle.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/digital-ocean.png[DigitalOcean, width=30%, link="https://www.digitalocean.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/osuosl.png[OSU Open Source Labs, width=30%, link="https://osuosl.org/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/datadog.png[Datadog, width=30%, link="https://www.datadoghq.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/docker.png[Docker, width=30%, link="https://www.docker.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/pagerduty.png[PagerDuty, width=30%, link="https://www.pagerduty.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/fastly.png[Fastly, width=30%, link="https://www.fastly.com"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}

We also want to recognize and thank link:https://github.com/[GitHub], link:https://jfrog.com/[JFrog], link:https://www.atlassian.com/[Atlassian], link:https://www.linuxfoundation.org/[Linux Foundation], link:https://www.netlify.com/[Netlify], and link:https://1password.com/[1Password] for providing tools to track Jenkins development.

image:/images/sponsors/github.png[GitHub, width=30%, link="https://github.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/jfrog.png[JFrog, width=15%, link="https://jfrog.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/atlassian.png[Atlassian, width=40%, link="https://www.atlassian.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/1password.png[1Password, width=30%, link="https://1password.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/netlify.png[Netlify, width=30%, link="https://www.netlify.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/linux-foundation.png[Linux Foundation, width=27%, link="https://www.linuxfoundation.org/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}

Massive thanks to link:https://www.algolia.com/[Algolia] for providing site search for the primary Jenkins documentation and plugins sites.

image:/images/sponsors/algolia.png[Algolia, width=40%, link="https://www.algolia.com/"]

We also want to share deep gratitude for operational funding from link:https://cd.foundation/[CD Foundation], link:https://www.cloudbees.com/[CloudBees], link:https://aws.amazon.com/[AWS], and link:https://www.digitalocean.com/[DigitalOcean].
Without funding, the community would have less opportunities to work on and develop Jenkins.

image:/images/sponsors/cdf.png[CD Foundation, link="https://cd.foundation/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/cloudbees.png[CloudBees, link="https://www.cloudbees.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/aws.png[AWS, link="https://aws.amazon.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
image:/images/sponsors/digital-ocean.png[DigitalOcean, width=30%, link="https://www.digitalocean.com/"]{nbsp}{nbsp}{nbsp}{nbsp}{nbsp}
```
Brand name

NotMyFault commented:
The open graph image mangles a good amount of company logos:

![Screenshot 2022-11-24 at 18 37 34](https://user-images.githubusercontent.com/13383509/203842001-37d10fcd-8941-446d-9572-5469e28b9f38.png)

I don't think this is a good representation for embeds.

lemeurherve commented on content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc:
```diff
@@ -0,0 +1,56 @@
+authors:
+- kmartens27
+description: >
+  The Jenkins project shares gratitude to our amazing sponsors.
```
```suggestion
  The Jenkins project shares gratitude to our amazing sponsors!
```
(nit)

lemeurherve reviewed (approved):
LGTM!

NotMyFault reviewed (changes requested):
https://github.com/jenkins-infra/jenkins.io/pull/5729#issuecomment-1326732054

Per above

kmartens27 commented:
I agree @NotMyFault. I'm away from my computer right now and wouldn't be able to generate that until much later today/tomorrow morning.

lemeurherve commented:
@NotMyFault @kmartens27 not perfect but I've got rid of the most glaring defects (1Password & Atlassian): [`876c6e4` (#5729)](https://github.com/jenkins-infra/jenkins.io/pull/5729/commits/876c6e4d4773e2263027724018f790498160fc82)

NotMyFault commented:
> @NotMyFault @kmartens27 not perfect but I've got rid of the most glaring defects (1Password & Atlassian): [`876c6e4` (#5729)](https://github.com/jenkins-infra/jenkins.io/pull/5729/commits/876c6e4d4773e2263027724018f790498160fc82)

Awesome, thanks! CDF's tagline is not complete still :/

lemeurherve commented:
> CDF's tagline is not complete still :/

D'oh, how I missed that... Fixed

NotMyFault reviewed (approved):
Nice

lemeurherve commented:
TIL about the "Swipe" and "Onion Skin" comparison options for images on GitHub (below the displayed image) 👀  https://github.com/jenkins-infra/jenkins.io/compare/a5b0908480102cf78b9681925558b1ead6c13c30..6a1ef01f0f1c2784d69bd3b389938533399d0bc4

kmartens27 commented:
Thank you so much for your help with this @lemeurherve and @NotMyFault!

backlogAK commented:
I checked and I think cloudbees logo on the site is out of proportion.
![image](https://user-images.githubusercontent.com/94915531/203975890-71066943-25ce-49c1-b2b6-23ec0f4683ce.png)

lemeurherve commented:
I tried to [use a fixed height instead of a % width for the images](https://github.com/jenkins-infra/jenkins.io/pull/5736) but gave up as it would need more case per case adjustments that I'm willing to spend time on.
